Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -1146,55 +1256,69 @@ def get_user_tokens(user_id):
     return result.get('items', [])
 
 
-def search_all_mailboxes(receive_only_accounts):
-    users_next_page_token = None
-    service = get_service('admin', 'directory_v1')
-    while True:
-        command_args = {
-            'maxResults': 100,
-            'domain': ADMIN_EMAIL.split('@')[1],  # type: ignore
-            'pageToken': users_next_page_token
-        }
+def search_in_mailboxes(list_accounts: list[str], receive_only_accounts: bool) -> None:
```

Comment:
1. Please add docstrings
2. If `list_accounts` is a list of accounts, a better name would be `accounts`.
3. `receive_only_accounts` is unclear